### ISSUE TITLE
Symfony 4 - encrypt:genkey/encrypt_key

### ISSUE THREAD

**Issue by Volmarg** [mogilvie/EncryptBundle#8]
Hello, me again.

**Symfony version: 4.2.x**

So before i forget or anything. Since I want to make my MIT project public later (where Your package is a part of it).

After installing Your package there are 3 problems.
* **Once package is installed it breaks the page**

*I'm not having screenshots or anything, but when Your install the package and without doing anything - refresh the page You get an exception that **encrypt_key** was not found*

I know You got to provide one by generating it via console but here actually comes next problem:

* **Cannot generate key before fixing the broken page - look above**

*The **encrypt:genkey** command is wont work untill the **encrypt_key** will be added in yaml*

* **Encrypt command is not working in CLI**

*(After fixing) When i try to generate key via **bin/console encrypt:genkey*** I get:
![image](https://user-images.githubusercontent.com/37488679/59827852-f2f3e580-9339-11e9-89c7-8c9e30dc7bc9.png)

*Summary*

* I cannot make the page run because i don't have the key
* I cannot make the key because the page is not working
* After all I cannot run the cli command to generate key
* What's more online generators that generate 256bit keys are kinda incompatibile (keept getting exceptions about to short/to long key)

*I've generally made my page run again but just saying that there are some problems*

**Comment by mogilvie** [comment]
Thanks @Volmarg.
I'm not running anything in Symfony 4 yet, so haven't installed it in that environment.
Please feel free to fork and update if you have time or inclination.
Cheers,
Mark

**Comment by Volmarg** [comment]
Sure, with some more time i will take a look since i need to have this working anyway. :)

**Comment by Volmarg** [comment]
I don't know how pulling/pushing here in github works for other ppl so i just gonna type some stuff in here. I work with gitlab without problem etc. ut got no idea how to do this fast here.

*EncryptorFactory.php*
Just changed this:
`public function createService($method, $encryptKey = null) `

And without adding parameters to yml it seems that it's working on brand fresh project **4.3** - will test that on mine later on, as i got oly acces to WIN OS atm.

![image](https://user-images.githubusercontent.com/37488679/59961367-fe304800-94d6-11e9-9ae2-590bd1aeb183.png)

That would fix the problem with missing param but I dont believe it will fix missing namespace in my project even if it works on windows atm.

----
<1>
Nvm. This doesn't help on my symfony 4.2 anyway

----
<2>
Ok, this is better for problem with missing: **encrypt_key**
**services.yml**
````
    SpecShaper\EncryptBundle\Encryptors\EncryptorService:
        factory: 'SpecShaper\EncryptBundle\Encryptors\EncryptorFactory:createService'
        arguments:
            - '%spec_shaper_encrypt.method%'
            - '@parameter_bag'
````
**EncryptorFactory**

````
    public function createService($method, $containerBag)
    {
        $encryptKey = '';
        $encryptKeyStringName = 'encrypt_key';

        if ($containerBag->has($encryptKeyStringName)) {
            $encryptKey = $containerBag->get($encryptKeyStringName);
        }

        switch($method){
            default:
                $encryptor = new OpenSslEncryptor($encryptKey);
        }
````

**Comment by mogilvie** [comment]
Fixed in version v1.1.1
